boards/arm/nucleo_f767zi/nucleo_f767zi.dts

```diff
@@ -112,3 +113,59 @@
 &rng {
 	status = "okay";
 };
+
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		/*
+		 * The two sectors 0-1 (32+32 kbytes) are reserved for
+		 * the bootloader.
+		 */
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x0 0x00010000>;
+			read-only;
+		};
+
+		/*
+		 * Sectors 2-3 (32+32 kbytes) unallocated.
```

Comment:
Can these 2 sectors be used for storage? This will allow to run NVS.

Reply:
done!